sqlglot/schema.py

```diff
@@ -308,6 +308,22 @@ def get_column_type(
 
         return exp.DataType.build("unknown")
 
+    def has_column(
```

Comment:
why don't you define __contains__ instead?

Reply:
you'd have to pass in an ast node or parse it

Reply:
chatted in Slack - this doesn't seem as flexible because the schema kicks off normalization automatically and so the burden of normalizing a table / column would be transferred to the user